Tasks/AzureMysqlDeployment/azureMysqlDeploy.ts

```diff
@@ -0,0 +1,63 @@
+import task = require('vsts-task-lib/task');
+import Q = require('q');
+import path = require('path');
+import { FirewallRule, FirewallAddressRange } from './models/Firewall';
+import { MysqlServer } from './models/MysqlServer';
+import { AzureMysqlTaskParameter } from './models/AzureMysqlTaskParameter';
+import { FirewallOperations } from './operations/FirewallOperations';
+import { MysqlServerOperations } from './operations/MysqlServerOperations';
+import { ToolPathOperations } from './operations/ToolPathOperations';
+import { ISqlClient } from './sql/ISqlClient';
+import { MysqlClient } from './sql/MysqlClient';
+import { AzureRMEndpoint } from 'azure-arm-rest/azure-arm-endpoint';
+import { AzureEndpoint } from 'azure-arm-rest/azureModels';
+import { ApplicationTokenCredentials } from 'azure-arm-rest/azure-arm-common';
+
+async function run() {
+    let azureMysqlTaskParameter: AzureMysqlTaskParameter;
+    let firewallAdded: boolean;
+    try {
+        task.debug('Task execution started');
+        task.setResourcePath(path.join( __dirname, 'task.json'));
+        // Get all task input parameters
+        azureMysqlTaskParameter = new AzureMysqlTaskParameter();
+        task.debug('parsed task inputs');
+        const endpoint: AzureEndpoint = await new AzureRMEndpoint(azureMysqlTaskParameter.getConnectedServiceName()).getEndpoint();
+        if(!endpoint){
```

Comment:
We can have empty lines before and after if/for blocks for better readability.

Reply:
Added after if block.